modules/user-access/pages/user-management.adoc

```diff
@@ -268,6 +264,162 @@ GSQL > REVOKE ROLE role1, role2 ON GRAPH example_graph
 
 The above command will revoke roles `role1` and `role2` on graph `example_graph` from users `user1` and `user2`.
 
+== Grant privileges to a user
+
+Users with the `WRITE_USER` privileges on global can grant RBAC privileges to the users.
```

Comment:
Need to clarify better: Only users with the WRITE_USER privilege at the global level can grant RBAC privileges to other users.

Reply:
Ditto.